### DISCUSSION TITLE
Feature Request: Please make Nova uses GraphQL instead REST to access backend API.

### DISCUSSION THREAD

**Discussion by wemersonrv** [laravel/nova-issues#4457] [Ideas & Feature Requests]
Hello,

It's possible to implement a way to make nova change the way it gets data from backend?

How about change from REST to GraphQL ? Or keep current wa and add option to change to GraphQL client.

Nova with [Hasura.io](https://hasura.io/) (GraphQL + Postgree + Realtime) would be a great teamup.

**Comment by nibri10** [comment]
Hello,
It would be great if this is possible mainly because of the performance that will be employed, and there will be a reduction in the use of server resources in some recurring tasks like CRUD.

**Comment by davidhemphill** [comment]
Currently no plans to switch the entire backend store for Nova.

**Comment by dillingham** [comment]
@davidhemphill might be cool to offer graphql as a separate access point for devs. AWA Amplify does this. Add your admin panel and immediately have an api. I could see this being a huge selling point for mobile app developers. 

Dynamic generation of schemas for lighthouse I presume. Not really familiar with the setup
